pkg/logging/logger.go

```diff
@@ -47,6 +50,22 @@ func Init(verbosity Verbosity) {
 		logger = *NewDefaultLoggerWithVerbosity(verbosity)
 	})
 }
+
+// Tee() redirect the output into the default log, and a file
+func Tee(fileName string) error {
```

Comment:
can also add this option through the `Init()` function?

Reply:
not a natural software engineering:
init is called once, even before we reads the args, and therefore do not know the log file.
Tee might be called more than once per run

Reply:
the args are available before the call to `init`